skimage/metrics/__init__.py

```diff
@@ -1,6 +1,11 @@
 from ._adapted_rand_error import adapted_rand_error
 from ._variation_of_information import variation_of_information
 from ._contingency_table import contingency_table
+from .overlap import (
+        BoundingBox,
+        intersect,
+        intersection_over_union,
+        )
```

Comment:
```suggestion
        disjoint,
        )
```
`disjoint` is public now :slightly_smiling_face: